src/cli/__tests__/getConfiguration.test.js

```diff
@@ -18,18 +17,6 @@ test('without template throws', async () => {
   );
 });
 
-test('with template transforms to its relative path', async () => {
```

Comment:
The generation of other apps breaks on my machine now.

Reply:
fixed in 48c3905